src/main/java/hw6/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,47 @@
+package hw6.pages;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+@Url("/metals-colors.html") @Title("Metal and Colors")
+public class MetalsAndColorsPage extends WebPage {
```

Comment:
I expect that you created a structer of PO similar to this: Site -> Page -> Form::submit(Entity e)
Check login form out and reveal Entity Driving approach

Reply:
fixed